### PR TITLE
Develop/community

### LINKED ISSUE #9: Display community Articles on the community list
When the user clicks the "Community" tab, no articles are shown on that screen. We should show the articles that exists in the Firestore Database.

### PR DESCRIPTION
**Pull request by veronica-hinojosa**
Closes #9

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -74,9 +74,13 @@ dependencies {
     implementation "androidx.activity:activity-compose:$activity_compose_version"
     implementation "com.google.accompanist:accompanist-pager:$view_pager_compose"
     implementation "com.google.accompanist:accompanist-pager-indicators:$view_pager_compose"
+//    implementation "androidx.paging:paging-compose:$paging_version"
```
Comment (ponchOlvera) [inline app/build.gradle:77]: If the dependency is not use it anymore, we can remove it from the file.

File: app/src/main/java/com/wizeline/mobilenews/domain/usecases/GetCommunityNewsUseCase.kt [hunk 2]
```diff
@@ -1,13 +1,18 @@
...
 
 interface GetCommunityNewsUseCase {
-    suspend operator fun invoke(
-        query: List<String> = emptyList(),
-        dateFrom: Long = System.currentTimeMillis(),
-        dateTo: Long? = null,
-        pageSize: Int = 20,
-        page: Int = 1
-    ): List<CommunityArticle>
+//    suspend operator fun invoke(
```
Comment (ponchOlvera) [inline app/src/main/java/com/wizeline/mobilenews/domain/usecases/GetCommunityNewsUseCase.kt:9]: Just remove the commented code here

File: app/src/main/java/com/wizeline/mobilenews/domain/usecases/GetCommunityNewsUseCaseImpl.kt [hunk 3]
```diff
@@ -9,12 +14,22 @@ import javax.inject.Inject
...
 ) : GetCommunityNewsUseCase, NewsDataToNewsDomain<CommunityArticle, Article>() {
-    override suspend fun invoke(
-        query: List<String>,
-        dateFrom: Long,
-        dateTo: Long?,
-        pageSize: Int,
-        page: Int
-    ): List<CommunityArticle> =
-        communityRepository.searchNews(query, dateFrom, dateTo, pageSize, page)
+//    override suspend fun invoke(
```
Comment (ponchOlvera) [inline app/src/main/java/com/wizeline/mobilenews/domain/usecases/GetCommunityNewsUseCaseImpl.kt:17]: Remove this as well, if we need it later we can get it through the tree history.

File: app/src/main/java/com/wizeline/mobilenews/ui/dashboard/CommunityViewModel.kt [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+@ExperimentalCoroutinesApi
+@HiltViewModel
+class CommunityViewModel @Inject constructor(private val useCase: GetCommunityNewsUseCase) : ViewModel()  {
+
+//    fun getArticles() {
+//        viewModelScope.launch {
+//            val results = useCase()
+//            println("COMMUNITY RES: $results")
+//        }
+//    }
```
Comment (ponchOlvera) [inline app/src/main/java/com/wizeline/mobilenews/ui/dashboard/CommunityViewModel.kt:24]: And this one as well.

**Review by ponchOlvera — APPROVED** [review]
Just small comments about the commented code. The articles look great in the app, nice implementation!

**Comment by veronica-hinojosa** [comment]
Oops, sorry, I left all the comments

**Comment by veronica-hinojosa** [comment]
Closes #9

**Pull request merged**